### ISSUE TITLE
Only show when full page refresh

### ISSUE THREAD

**Issue by MogyahidAnsid** [php-flasher/php-flasher#155]
I am using livewire V3. notyf only shows when full refreshing the page. But triggering on dispatch listener doesn't show.
If I refresh the parent component, the previous notyf will pop-up.

```
    // Modal Function
    public function save()
    {
        $this->dispatch('project-created');
    }


   // Parent livewire component
     #[On('project-created')]
    public function projectCreated()
    {
        notyf()->duration(2000)
            ->dismissible(true)
            ->ripple(false)
            ->addSuccess('Project saved successfully!');
    }
```

**Comment by Mikhail-Konstantinou** [comment]
Indeed, I experience the same error now that I upgraded to livewire 3. I tried to follow the documentation for a brand new livewire 3 project and the steps in the Livewire integration page... and it does not work as it works in Livewire 2. 

Did you find any workaround to this?

**Comment by FernandoRub13** [comment]
The same here :(

**Comment by akinsayo** [comment]
Same issue..

**Comment by codebyshami** [comment]
Have you guys found it out how to do it or not?

**Comment by yoeunes** [comment]
Hello guys,

I've just released PHPFlasher v1.15.5 with support for Livewire v3. It's not fully tested yet, so I'd appreciate if you could test it in your environment. Please update using:

```bash
composer require php-flasher/flasher-laravel ^1.15.5
```

Let me know if there are any issues. Your feedback will help in deciding whether to close this issue.

Thanks!
Younes

**Comment by akinsayo** [comment]
You just made my life easier !
This now works as expected.

It should be noted that the update instruction failed.
What worked was a composer update

Thank you !
